org.eclipse.jdt.ls.tests/projects/maven/aspose/pom.xml

```diff
@@ -0,0 +1,176 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"

```

Comment:
the build, reporting and profiles sections are not necessary

Reply:
Fixed.